<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewStockDialog</class>
 <widget class="QDialog" name="ViewStockDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stock</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableWidget" name="tabel_widget_stock"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btn_ok">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
